charger/prachtalpha.go

```diff
@@ -30,17 +30,18 @@ import (
 
 // PrachtAlpha charger implementation
 type PrachtAlpha struct {
-	conn    *modbus.Connection
-	vehicle uint16
-	curr    uint16
+	conn              *modbus.Connection
+	vehicle           uint16
+	curr              uint16
+	currentWorkaround bool
```

Comment:
Besser als Automatik abhängig von den Firmware-Versionen?

Reply:
Ja, das kann man nicht automatisch entscheiden. Eigentlich kann man jetzt nur den vorherigen Workaround ausschalten, weil der eigentlich nicht richtig ist.

Reply:
Woher soll ein Anwender verstehen und wissen was er da einstellen soll?
Das kann IMHO nur irgendwie automatisch in der Implementierung getestet und entschieden werden. Wie in anderen Fällen auch.

Die Anwender sind mit sowas kryptischem regelmäßig überfordert -> erhöhte Nachfragen und suboptimales Ladeerlebnis. Kurz: unschön.

Reply:
Es gibt auch einen Pull Request in evcc-io/docs 

Wie soll der Anwender überhaupt irgendwas entscheiden? Im Endeffekt, wenn Anwender nicht einstellen, dann ist es so, wie es aktuell eh ist. Und wenn Anwender Probleme haben, dann haben sie jetzt die Möglichkeit, den ursprünglichen Workaround rückgängig zu machen.